Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/NativeApp/AndroidBasicInteractionsTest.cs

```diff
@@ -44,6 +44,41 @@ public void AlertInteraction()
             alertDialog.Close();
         }
 
+        [Test]
+        public void TestVerticalSwipeToElement()
+        {
+            var viewControlsScreen = new ViewControlsScreen();
+            OpenRadioButtonsScreen();
+            viewControlsScreen.ScrollToAllInsideScrollViewLabel();
+            Assert.AreEqual(
+                viewControlsScreen.AllInsideScrollViewLabelText,
+                "(And all inside of a ScrollView!)",
+                "Label text does not match expected");
+            viewControlsScreen.ScrollToDisabledButton();
+            Assert.IsFalse(viewControlsScreen.IsDisabledButtonClickable);
+        }
+
+        [Test]
+        public void TestHorizontalSwipeToElement()
+        {
+            var friendlyMessage = "Tab text does not match expected";
+            var viewTabsScrollableScreen = new ViewTabsScrollableScreen();
+            OpenViewTabsScrollableScreen();
+            Assert.IsTrue(viewTabsScrollableScreen.State.IsDisplayed);
+            viewTabsScrollableScreen.SwipeTab(4, 1);
+            viewTabsScrollableScreen.SelectTab(7);
+            Assert.AreEqual(
+                viewTabsScrollableScreen.GetTabContentText(7),
+                "Content for tab with tag Tab 7",
+                friendlyMessage);
+            viewTabsScrollableScreen.SwipeTab(5, 7);
+            viewTabsScrollableScreen.SelectTab(4);
+            Assert.AreEqual(
+                viewTabsScrollableScreen.GetTabContentText(4),
+                "Content for tab with tag Tab 4",
```

Comment:
You could probably move these `4` and `7` into local variables like you did with `friendlyMessage`